llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3649,11 +3649,10 @@ void LoopVectorizationCostModel::collectLoopUniforms(ElementCount VF) {
     if (IsOutOfScope(V))
       continue;
     auto *I = cast<Instruction>(V);
-    auto UsersAreMemAccesses =
-      llvm::all_of(I->users(), [&](User *U) -> bool {
-        auto *UI = cast<Instruction>(U);
-        return TheLoop->contains(UI) && IsVectorizedMemAccessUse(UI, V);
-      });
+    bool UsersAreMemAccesses = llvm::all_of(I->users(), [&](User *U) -> bool {
```

Comment:
while you are editing the line, might as well drop redundant llvm::


```suggestion
    bool UsersAreMemAccesses = all_of(I->users(), [&](User *U) -> bool {
```